src/test/java/de/tum/in/www1/artemis/StudentExamIntegrationTest.java

```diff
@@ -1108,7 +1110,8 @@ private void assertVersionedSubmission(Submission submission) {
             assertThat(((TextSubmission) submission).getText()).isEqualTo(versionedSubmission.get().getContent());
         }
         else if (submission instanceof ModelingSubmission) {
-            assertThat(((ModelingSubmission) submission).getModel()).isEqualTo(versionedSubmission.get().getContent());
+            ModelingSubmission modelingSubmission = (ModelingSubmission) submission;
+            assertThat("Model: " + modelingSubmission.getModel() + "; Explanation: " + modelingSubmission.getExplanationText()).isEqualTo(versionedSubmission.get().getContent());
```

Comment:
As the line is a little too long, maybe it could be converted to a multi-line statement just to keep consistency 😃

Reply:
I also had it as a multi-line statement, but the server-style check pointed out to change it to a single-line statement